src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -20,7 +20,7 @@ import {
 import { datasetBuilderCohorts } from 'src/pages/library/datasetBuilder/state';
 
 describe('CohortEditor', () => {
-  const datasetDetails = dummyDatasetDetails;
+  const datasetDetails = dummyDatasetDetails('axin');
```

Comment:
Adding a parameter is OK but maybe use a string like `unused` instead of `axin` so it's clear that the value is ignored by the test. I think I did it this way because, until we have some Ajax implemented (or more complex mocks), the id is unused in the code.